variables.tf

```diff
@@ -344,6 +344,36 @@ variable "hms_ro_ecs_task_count" {
   default     = "3"
 }
 
+variable "hms_rw_k8s_replica_count" {
+  description = "Initial Number of read/write Hive Metastore k8s pod replicas to create."
+  type        = number
+  default     = 3
+}
+
+variable "hms_ro_k8s_replica_count" {
```

Comment:
Maybe adding `hms_ro_k8s_min_replica_count` instead of `hms_ro_k8s_replica_count`